### DISCUSSION TITLE
Questions, Suggestions, Anything

### DISCUSSION THREAD

**Discussion by theodorusclarence** [theodorusclarence/ts-nextjs-tailwind-starter#75] [General]
Hey, you can leave some questions, improvements, appreciation, or anything here.

Tell me what's your favorite feature in the starter?

**Comment by Isaac-Tait** [comment]
Wow! Really cool starter. Thank you for putting this out there. Might I suggest adding `yarn install` to Step 1 in the ReadME? Some folks may not recognize they need to do that and then be confused by the errors right at the start. Great work though 💯

**Reply by theodorusclarence** [reply]
Hey, that's a great idea. 

Thanks for the suggestions, I'll add them soon.

**Comment by osseonews** [comment]
Thanks so much for this starter. Amazing work.

**Comment by sykespro** [comment]
Any plans to update the starter kit to use Next.js 13?

**Reply by theodorusclarence** [reply]
Hi, the starter is already on next 13, but I decided not to include the new app directory layout because it’s still in a beta.

I’ll move the starter to the app directory until the stable release.

**Comment by cha2hyun** [comment]
Hello. I'm working on a project using `use this template before, and I'm wondering how I can upgrade to the latest commit (next.js 13) and expansion pack. 

Anyway, thanks for amazing starter and expansion pack

**Reply by theodorusclarence** [reply]
Hi, I believe you can follow the guide on [this page](https://nextjs.org/blog/next-13)

for the expansion pack, there is no way to migrate it with CLI, so you need to copy and paste the changes manually

**Comment by Johnrobmiller** [comment]
I noticed that the `Seo.tsx` component was removed and that all those really good `<Head />` defaults were removed with it. What was the reason behind that decision?

**Reply by theodorusclarence** [reply]
Yes it’s gone, check out this issue

https://github.com/theodorusclarence/ts-nextjs-tailwind-starter/issues/243
